packages/browser-agent-core/src/features/page-view-timing/instrument/long-tasks.js

```diff
@@ -0,0 +1,51 @@
+import { subscribeToEOL } from "@newrelic/browser-agent-core/src/common/unload/eol";
+
+export const onLT = (onReport) => {
```

Comment:
This could really benefit from jsdoc, as all these comments could come through with intellisense

Reply:
Can we rename this function to `onLongTask`